BankManagerConsoleApp/BankManagerConsoleApp/CustomerQueue.swift

```diff
@@ -9,6 +9,10 @@ import Foundation
 
 struct CustomerQueue<T> {
     private var list: LinkedList<T>
+    
+    init(list: LinkedList<T>) {
```

Comment:
```suggestion
    init(list: LinkedList<T> = LinkedList<T>()) {
```

=> 초기에 어떤 값도 넣어주지 않는다면 [Default Parameter Value](https://docs.swift.org/swift-book/LanguageGuide/Functions.html#ID169) 를 집어넣어도 될 것 같아요 :)

Reply:
이렇게 함으로써 `CustomerQueue<Int>()` 만으로도 큐를 만들 수 있겠군요!